source/includes/geospatial-places-data-setup.rst

```diff
@@ -0,0 +1,26 @@
+Create a collection ``places`` with the following documents:
+
+.. code-block:: javascript
+
+   db.places.insert( {
+       name: "Central Park",
+      location: { type: "Point", coordinates: [ -73.97, 40.77 ] },
+      category: "Parks"
+   } );
+   db.places.insert( {
+      name: "Sara D. Roosevelt Park",
+      location: { type: "Point", coordinates: [ -73.9928, 40.7193 ] },
+      category: "Parks"
+   } );
+   db.places.insert( {
+      name: "Polo Grounds",
+      location: { type: "Point", coordinates: [ -73.9375, 40.8303 ] },
+      category: "Stadiums"
+   } );
```

Comment:
Two things here:

- `insert` is deprecated in mongosh. I just tried running this in mongosh and received deprecation warnings. We should always be using `insertOne` or `insertMany`
- Why are we using three separate insert statements instead of one insertMany statement?

Reply:
Good point, I was recycling what was already on the page but might as well update this while at it.

Reply:
Updated the include to be a insert many, rendering looks good : https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCS-14900-geoNear-Page-lacks-data-setup-for-commands-referenced/geospatial-queries/#examples

Reply:
@jeff-allen-mongo